java/ql/src/semmle/code/java/frameworks/android/Intent.qll

```diff
@@ -52,3 +52,13 @@ class BundleGetterMethod extends Method, TaintPreservingCallable {
 
   override predicate returnsTaintFrom(int arg) { arg = -1 }
 }
+
+/** A reader method on `android.os.Parcel`. */
+class ParcelReaderMethod extends Method, TaintPreservingCallable {
```

Comment:
Do Parcels relate to Intents particularly? Note if this moves elsewhere, any file that defines a `TaintPreservingCallable` should be imported by `FlowSteps.qll`'s private module `Frameworks`

Reply:
I've moved <code>ParcelReaderMethod</code> together with <code>BundleGetterMethod</code> from <code>Intent.qll</code> to <code>Android.qll</code>. The rationale is that their <code>android.os</code> package is a core package that doesn't need a separate library file just for these two methods. Please let me know if you want to move them to somewhere else or create a separate library.